### ISSUE TITLE
Signor processor error

### ISSUE THREAD

**Issue by kkaris** [sorgerlab/indra#1419]
When running `sources.signor.api.process_from_web` an `InvalidIdentifier` error is raised:

```text
InvalidIdentifier: Invalid identifier: SID:135317436 for PUBCHEM pattern ^\d+$
```

To re-create the error:
```python
from indra.sources.signor import process_from_web
sp = process_from_web()
```

**Comment by cthoyt** [comment]
PubChem has two databases - compounds and substances.  SID would correspond to a substance ID, which with some api calls can probably be resolved to a proper CID (compound ID)

**Comment by kkaris** [comment]
Resolved by #1423
